File: js/0403.js
// ŠKæ‚ğ‹‚ß‚éŠÖ”
function Fact(n) {
    if (n <= 1)
        return 1
    else
        return n * Fact(n - 1)
}

// 10‚ÌŠKæ
WScript.Echo(Fact(10))
WScript.Echo(10 * 9 * 8 * 7 * 6 * 5 * 4 * 3 * 2 * 1)

File: py/0403.py
# coding: Shift_JIS

# 階乗を求める関数
def Fact(n):
    if (n <= 1):
        return 1
    else:
        return n * Fact(n - 1)

# 10の階乗
print Fact(10)
print 10 * 9 * 8 * 7 * 6 * 5 * 4 * 3 * 2 * 1
